### ISSUE TITLE
Government Shutdown Banner

### ISSUE THREAD

**Issue by aboydnw** [NASA-IMPACT/veda-ui#1349]
### Context

We’d like to prep PRs to push out an announcement banner in the event of a government shutdown. These banners will then be pushed to production on Monday morning, if a shutdown occurs. But, we want to be ready because there will be no production pushes allowed after that time, until the US government is funded. 

The four instances we are looking to add a banner are:

- GHGC Portal
- earth.gov
- earth.gov/mobile-climate-mapper
- Earthdata***** text will be different as this is a NASA only site

### Acceptance Criteria

- [ ] Banners are live on production for GHGC and earth.gov with text to be approved by stakeholders (TBD)
- [ ] The expiration date for those banners should be June 1, 2025 (seems like a safe bet that the US government will be back online by then, then we will have another production push to remove the banner when that happens)

**Comment by sandrahoang686** [comment]
@aboydnw what should the banners content/description say?

**Comment by aboydnw** [comment]
Brian is working on getting final content. My guess is we won't have this until late today or early Monday morning. I just wanted to get this ticket created because we will probably have a very short time between getting final content and needing to push it to production

**Comment by aboydnw** [comment]
@sandrahoang686 I added the earthdata instance to the list here. we weren't expecting it originally but sounds like ESDS might request their sites to follow the same guidance as earth.gov sites. TBD

**Comment by freitagb** [comment]
made some edits to the top level comment to include earth.gov/mobile-climate-mapper and to note that the text will be different for the earthdata instance as it is a NASA only site.

**Comment by freitagb** [comment]
> Brian is working on getting final content. My guess is we won't have this until late today or early Monday morning. I just wanted to get this ticket created because we will probably have a very short time between getting final content and needing to push it to production

This is correct. Final text will not be available until Monday. We can have the PR ready with a header and a one sentence Lorem Ipsum if that works

**Comment by aboydnw** [comment]
Government passed a funding bill at the last moment, closing as not planned
